src/data/worldpop_stratified.csv

```diff
@@ -0,0 +1,3624 @@
+GB_HEF,1092,4382,5435,4982,5927,6495,6631,6411,5876,6288,5605,6613,6363,5503,4855,4899,3555,5897,1148,4608,5746,5346,6396,6816,6469,6241,5682,6059,5925,6531,6297,5308,4711,4615,3202,3851
+GB_HIL,1747,7011,8707,7938,9581,10447,10336,9873,9160,9980,9030,10711,10288,8855,7973,8191,5845,9348,1846,7413,9234,8633,10191,10910,10684,10429,9386,9832,9471,10380,10024,8491,7376,7074,4997,6294
+GB_HLD,1366,5480,6875,6105,7196,8144,8375,8042,7514,7925,6985,8404,8012,6896,6065,6211,4605,7458,1442,5791,7144,6844,8255,8544,8050,7822,6978,7557,7472,8077,7861,6659,5929,5718,3868,4765
+GB_HMF,1142,4583,5774,5258,6626,7228,6570,6230,5892,6355,5848,6923,6706,5983,5331,5362,3841,5985,1193,4792,5887,5513,6226,6653,7092,6966,6162,6522,6177,6785,6497,5292,4644,4560,3206,4182
+GB_HNS,1697,6808,8456,7710,9309,10149,10035,9586,8895,9689,8767,10400,9991,8602,7744,7954,5676,9077,1793,7198,8965,8381,9891,10589,10376,10127,9113,9548,9197,10079,9733,8242,7160,6868,4852,6112
+GB_HPL,535,2146,2678,2487,2965,3294,3302,3208,2920,3113,2788,3296,3200,2721,2418,2417,1803,2905,568,2281,2828,2599,3103,3260,3149,3023,2772,2968,2890,3177,3034,2603,2292,2268,1525,1896
+GB_HRT,5911,23713,29445,27108,32640,36016,36538,34830,31668,33791,30464,35967,34204,29587,26718,27001,19772,31794,6206,24917,31041,28763,34024,35990,34334,33617,30860,33004,31911,35141,34279,28899,25029,24467,16784,20944
+GB_HRW,1545,6199,7699,7019,8472,9238,9138,8727,8099,8825,7985,9472,9098,7831,7050,7245,5169,8266,1633,6555,8165,7635,9013,9648,9450,9225,8301,8694,8375,9178,8864,7509,6522,6254,4419,5566
+GB_HRY,1696,6804,8526,7851,9892,10391,9442,9141,8473,9252,8730,10227,9967,8687,7818,7978,5682,8973,1768,7098,8765,8120,9164,10193,10817,10426,9401,9842,9101,10100,9609,8032,6975,6734,4768,6102
+GB_IOS,8,32,40,37,44,48,49,47,44,47,41,49,48,41,36,36,26,43,8,34,42,39,46,49,46,45,40,43,43,46,44,38,34,33,23,28
+GB_IOW,773,3102,3840,3516,4133,4595,4677,4550,4156,4454,3975,4705,4477,3912,3498,3522,2573,4164,812,3261,4074,3794,4589,4826,4596,4405,4024,4285,4186,4599,4483,3740,3273,3211,2210,2736
+GB_ISL,1341,5381,6748,6209,7832,8236,7466,7220,6704,7315,6902,8085,7883,6881,6190,6309,4493,7086,1398,5613,6927,6422,7239,8043,8557,8254,7432,7786,7200,7990,7599,6341,5509,5327,3772,4837
+GB_IVC,417,1674,2099,1924,2341,2563,2557,2485,2283,2473,2165,2612,2509,2138,1903,1956,1464,2363,438,1760,2173,2022,2367,2522,2448,2349,2132,2244,2240,2409,2327,1992,1752,1678,1118,1361
+GB_JSY,664,2557,2684,2880,3960,4514,4539,3233,2241,2975,2841,4086,3970,3397,2903,2671,1847,3242,704,2714,2908,3128,4153,4658,4655,3624,2417,2871,3028,3843,3822,3192,2516,2133,1332,1904
+GB_KEC,936,3755,4732,4309,5436,5931,5378,5098,4825,5202,4789,5670,5493,4906,4370,4393,3147,4901,977,3924,4819,4514,5090,5440,5813,5710,5049,5345,5060,5559,5321,4329,3801,3734,2625,3427
+GB_KEN,8952,35913,44792,41421,49630,54303,55273,53304,48421,51375,46140,54244,52603,45237,40538,40737,29754,48708,9450,37942,47068,43430,51612,55052,5
```

Comment:
Any idea why we're missing a lot of regions of Japan?  This leads to an odd mixture of data sources in demographics.csv

Reply:
This happens because the corresponding Wikidata entities do not have an Open Street Maps ID associated with them, for example: https://www.wikidata.org/wiki/Q1037393

I want to coordinate some volunteering efforts to update all the missing Wikidata items with the appropriate Open Street Maps ID, but we obviously can't wait for that to happen...